tests/main/parallel-install-classic/task.yaml

```diff
@@ -0,0 +1,88 @@
+summary: Checks parallel installation of classic snaps
+
+# core does not support classic confinement
+systems: [-ubuntu-core-*]
+
+prepare: |
+    # ensure we have no snap user data directories yet
+    rm -rf /home/test/snap
```

Comment:
I would be more comfortable with the test moving /home/test/snap and /root/snap aside instead.